docker-compose.yml

```diff
@@ -0,0 +1,96 @@
+version: '3'
+
+volumes:
+  streaming:
+  database:
+  fedora:
+  work:
+  solr:
+
+networks:
+  external:
+  internal:
+    
+services:
+  db:
+    image: avalonmediasystem/db:fedora4
+    volumes:
+      - database:/data
+    environment:
+      - AVALON_DB_PASSWORD
+      - FEDORA_DB_PASSWORD
+      - PGDATA=/data
+      - POSTGRES_USER=postgres
+    networks:
+      internal:
+  fedora:
+    image: avalonmediasystem/fedora:4.7.4
+    depends_on:
+      - db
+    volumes:
+      - fedora:/data
+    environment:
+      - JAVA_OPTIONS=${JAVA_OPTIONS} -Dfcrepo.postgresql.password=${FEDORA_DB_PASSWORD} -Dfcrepo.postgresql.username=fedora -Dfcrepo.postgresql.host=db -Dfcrepo.modeshape.configuration=classpath:/config/jdbc-postgresql/repository.json
+    networks:
+      internal:
+  solr:
+    image: avalonmediasystem/solr:latest
+    volumes:
+      - solr:/opt/solr/server/solr/mycores
+    entrypoint:
+      - docker-entrypoint.sh
+      - solr-precreate
+      - avalon
+      - /opt/solr/avalon_conf
+    networks:
+      internal:
+  matterhorn:
```

Comment:
Please remove this since we're going to start without matterhorn.

Reply:
✅